justfile

```diff
@@ -2,7 +2,7 @@ targets := "arm-unknown-linux-gnueabi armv7-unknown-linux-gnueabihf x86_64-unkno
 
 target_dir := "target"
 
-# list of all features except criterion
+# list of all features
 all_features := "nalgebra_support fixed"
```

Comment:
We could probably just use the `--all-features` flag here now I think?

Reply:
I think so.